### PR TITLE
Add time picker

### LINKED ISSUE #16: Add igTimePicker to toolbox

### PR DESCRIPTION
**Pull request by Lipata**
Closes #16 

@ig-ealbert We will need translations for the new options, especially for the igTimePicker controls. If you want I can put them, then you will need only the check their correctness.

@sdimchevski [here](http://designer.dev.igniteui.local/) you can see that all the igDatePicker, igDateEditor and igTimePicker has the same icon in the toolbox at the right. If we can create two separate icons for igDataEditor and igTimePicker that will be great.

### CODE REVIEW

File: src/config/dateeditor.json [hunk 1]
```diff
@@ -314,13 +314,70 @@
...
+              "en": "Value for the seconds",
+              "jp": "Value for the seconds"
+          }
+        },
+        "milliseconds": {
+            "type": "number",
+            "defaultValue": 1,
+            "description": {
+                "en": "Value for the minutes",
+                "jp": "Value for the minutes"
```
Comment (ig-ealbert) [inline src/config/dateeditor.json]: @Lipata Is this correct?  Should it be "Value for the milliseconds"?

File: src/config/datepicker.json [hunk 2]
```diff
@@ -314,13 +314,70 @@
...
+          "description": {
+              "en": "Value for the seconds",
+              "jp": "Value for the seconds"
+          }
+        },
+        "milliseconds": {
+            "type": "number",
+            "defaultValue": 1,
+            "description": {
+                "en": "Value for the minutes",
```
Comment (ig-ealbert) [inline src/config/datepicker.json]: Same here.

File: src/config/dateeditor.json (hunk 1 above)
Comment (Lipata) [reply src/config/dateeditor.json]: Yes, thanks. I will correct both.

**Comment by ig-ealbert** [comment]
@Lipata I will add the translations now.  This will take a while as I'm doing it by text format with copy/paste.  Additionally, since @yoshimis is out of the office until next week, these are essentially draft translations.  Most of them were 100% matches with the Ignite UI client-side APIs so we should be good on those, but it is possible that we will change them in a week.

**Pull request merged**